### ISSUE TITLE
Request: Shouldn't require much assistance (almost zero assistance), with reference to using the full shape, expression model.

### ISSUE THREAD

**Issue by drtcp** [1adrianb/face-alignment#363]
Hello, excellent paper excellent repository.

I have seen a few requests for the full model, rather than 62 parameters (12 pose, 40 shape, 10 expression).
Can you provide the dull 3D morphable model (12 pose, 199 shape and 29 expression, or is it available via the BFM model site)?

If you can point me at (or provide) the full model I might be able to provide additional training code to fit the full model (or a user defined subset of it). 

I ask, because the paper shows some excellent inventive steps: use of projected cylindrical convolution & parameter fitting order. I believe that this could produce a significant increase in accuracy for face reconstruction; both shape and texture. 

I do not require much help (probably zero, but one never knows), just access to the data required to train the full model parameter fitting.

Can you provide any assistance?

**Comment by drtcp** [comment]
Please forgive this request, my mistake.
